parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -376,4 +376,44 @@ should_skip_nvidia_drivers() {
     should_skip=$(echo "$body" | jq -e '.compute.tagsList | map(select(.name | test("SkipGpuDriverInstall"; "i")))[0].value // "false" | test("true"; "i")')
     echo "$should_skip"
 }
+
+start_watch () {
+  capture_time=$(date +%s)
+  dt_start=$(date +%H:%M:%S)
+}
+
+stop_watch () {
+
+  local current_time=$(date +%s)
+  local dt_end=$(date +%H:%M:%S)
```

Comment:
nit: rename to end_timestamp, for clarity too too?

Reply:
Will do for both!